SWC compilation creates regression on 4.4.0

**Describe the bug**
After upgrading from 4.3.2 to 4.4.0, I can no longer import any API models. It appears that the generated `index.js` files are attempting to import `.js` files (i.e. `v1beta1.js`) which do not exist. I'm not exactly sure why this is the case, but it seems to have something to do with the fact that the models are now compiled using swc.

```
Error: Cannot find module './v1beta1.js'
Require stack:
- node_modules/kubernetes-models/admissionregistration.k8s.io/index.js
- node_modules/kubernetes-models/index.js
```

Could it be possible that in some implementations of import systems this bug can be handled and thus it slipped through?

**To Reproduce**
Attempt to import any model from `kubernetes-models` with an import statement.

**Expected behavior**
The module should have been imported.

**Additional context**
I am running my program with ts-node and swc with direct transpilation. I haven't tested any other methods.

--- a/.changeset/mean-years-add.md
+++ b/.changeset/mean-years-add.md
@@ -1,0 +1,6 @@
+---
+"@kubernetes-models/crd-generate": patch
+"@kubernetes-models/openapi-generate": patch
+---
+
+Append `/index` to generated paths to ensure correct import handling.

--- a/utils/crd-generate/src/generators/alias.ts
+++ b/utils/crd-generate/src/generators/alias.ts
@@ -18,7 +18,7 @@ function generate(map: KeyMap, parent = ""): OutputFile[] {
       content += `export * from "./${val}";\n`;
     } else {
       const exportedName = camelCase(key, ".-");
-      content += `export * as ${exportedName} from "./${key}";\n`;
+      content += `export * as ${exportedName} from "./${key}/index";\n`;
       children = children.concat(generate(val, parent + key + "/"));
     }
   }

--- a/utils/openapi-generate/src/generators/alias.ts
+++ b/utils/openapi-generate/src/generators/alias.ts
@@ -40,7 +40,7 @@ export default function ({ getDefinitionPath }: Context): Generator {
 
           output.push({
             path: aliasPath,
-            content: `export * from "${getRelativePath(aliasPath, defPath)}";`
+            content: `export * from "${getRelativePath(aliasPath, defPath)}/index";`
           });
         }
       }
@@ -74,7 +74,7 @@ export default function ({ getDefinitionPath }: Context): Generator {
           .map((v) => {
             const exportedName = camelCase(v, ".-");
 
-            return `export * as ${exportedName} from "./${v}";`;
+            return `export * as ${exportedName} from "./${v}/index";`;
           })
           .join("\n")
       });

